torch/onnx/_internal/fx/export.py

```diff
@@ -78,62 +189,37 @@ def _create_onnx_friendly_decomposition_table() -> Dict[
 _ONNX_FRIENDLY_DECOMPOSITION_TABLE = _create_onnx_friendly_decomposition_table()
 
 
-def _retrieve_or_wrap_scalar_as_constant(
-    g, fx_node_arg, fx_name_to_ts_value, example_output
-):
+def _retrieve_or_adapt_input(fx_node_arg, fx_name_to_onnxscipt_tensor, example_output):
     """Map FX value to TorchScript value.
 
     When creating TorchScript graph from FX graph, we need a mapping from FX variable
     to TorchScript variable. This function maps FX variable, fx_node_arg, to torch.jit.Value.
     """
+    del example_output  # unused
 
-    ts_value = fx_node_arg
-    if isinstance(ts_value, torch.fx.Node):
+    onnx_tensor = fx_node_arg
```

Comment:
`onnx_tensor` is confusing since it can be other types. Let's call it `onnxscript_value`.

Reply:
SG!

Reply:
TB updated